apps/shipments/models.py

```diff
@@ -341,3 +343,98 @@ def update_vault_hash(self, vault_hash, rate_limit=False):
     VALID_UNTIL = 24
     FUNDING_TYPE = FundingType.SHIP.value
     SHIPMENT_AMOUNT = 1
+
+
+class TrackingData(models.Model):
+    id = models.CharField(primary_key=True, default=random_id, max_length=36)
+    created_at = models.DateTimeField(auto_now_add=True)
+    device_id = models.CharField(blank=True, null=True, max_length=36)
+    shipment = models.ForeignKey(Shipment, on_delete=models.PROTECT, null=True)
+    latitude = models.FloatField(default=None)
+    longitude = models.FloatField(default=None)
+    altitude = models.FloatField(default=None)
```

Comment:
Don't set default=None for FloatField, that's not a valid attribute (and it shouldn't be nullable)